utils/include/detray/detectors/build_toy_detector.hpp

```diff
@@ -86,7 +86,7 @@ struct toy_det_config {
     // Radius of the innermost volume that contains the beampipe
     scalar_t m_beampipe_volume_radius{25.f * unit<scalar_t>::mm};
     // Envelope around the modules used by the cylinder portal generator
-    scalar_t m_portal_envelope{0.5f * unit<scalar_t>::mm};
+    scalar_t m_portal_envelope{0.1f * unit<scalar_t>::mm};
```

Comment:
Is this change necessary?

Reply:
Yes, the navigation is more precise with tighter portals